### PR TITLE
Adds /mlpol/ to PreMiD

### PR DESCRIPTION
**Pull request by Cheesegriller**
That's pretty much it.
/mlpol/ is a imageboard 
I was gonna have Ponerpics but Codacy wont let me breathe oxygen.

### CODE REVIEW

File: websites/M/MLPOL/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "StrangeUniverseArtist",
+    "id": "727029933255163946"
+  },
+  "url": "mlpol.net",
+  "description": {
+    "en": "Welcome to the Mongolian political horsewhispering forum, also known as /mlpol/"
+  },
+  "service": "mlpol",
```
Comment (Bas950) [inline websites/M/MLPOL/dist/metadata.json:11]: service name and folder name must be the same (caps inc.)

File: websites/M/MLPOL/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "version": "1.0.0",
+  "logo": "https://i.ibb.co/Hq6PWGv/mlpol.png",
+  "thumbnail": "https://i.ibb.co/rZJ31w7/thumbnail2.png",
```
Comment (Bas950) [inline websites/M/MLPOL/dist/metadata.json:14]: upload to imgur

File: websites/M/MLPOL/presence.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
+const browsingStamp = Math.floor(Date.now() / 1000), presence = new Presence({
+    clientId: "781944209770151997" //The client ID of the Application created at https://discordapp.com/developers/applications
+});
```
Comment (Bas950) [inline websites/M/MLPOL/presence.ts:3]: remove comment

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof of it working

File: websites/M/MLPOL/presence.ts (hunk 3 above)
Comment (Cheesegriller) [reply websites/M/MLPOL/presence.ts]: Done

File: websites/M/MLPOL/dist/metadata.json (hunk 2 above)
Comment (Cheesegriller) [reply websites/M/MLPOL/dist/metadata.json]: Done

File: websites/M/MLPOL/dist/metadata.json (hunk 1 above)
Comment (Cheesegriller) [reply websites/M/MLPOL/dist/metadata.json]: Done

**Comment by Cheesegriller** [comment]
image
proof it works

**Review by Bas950 — APPROVED** [review]
upload proof of it working

**Review by ririxidev — APPROVED** [review]

**Pull request merged**